fastcrypto-zkp/src/api.rs

```diff
@@ -31,8 +33,13 @@ pub fn verify_groth16_in_bytes(
     proof_public_inputs_as_bytes: &[u8],
     proof_points_as_bytes: &[u8],
 ) -> Result<bool, FastCryptoError> {
-    let x = BlsFr::deserialize_compressed(proof_public_inputs_as_bytes)
-        .map_err(|_| FastCryptoError::InvalidInput)?;
+    let mut x = Vec::new();
```

Comment:
We could also return fast if it's an empty public inputs. We won't fail, its possible for a circuit to not have public inputs, we need a test for that too.